pkg/provider/aws/knowledgebase/edge_expansion_test.go

```diff
@@ -177,6 +177,115 @@ func Test_ExpandEdges(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "single expose k8s",
+			edge: graph.Edge[core.Resource]{
+				Source:      &resources.RestApi{Name: "api"},
+				Destination: &resources.TargetGroup{Name: "tg"},
+				Properties: dgraph.EdgeProperties{
+					Data: knowledgebase.EdgeData{
+						AppName:     "my-app",
+						Source:      &resources.RestApi{Name: "api"},
+						Destination: &resources.TargetGroup{Name: "tg"},
+						Routes:      []core.Route{{Path: "/route/1", Verb: "post"}, {Path: "/route/1", Verb: "get"}, {Path: "/route/2", Verb: "post"}, {Path: "/", Verb: "get"}},
+					},
+				},
+			},
+			want: coretesting.ResourcesExpectation{
+				Nodes: []string{
+					"aws:api_integration:my-app-/-GET",
+					"aws:api_integration:my-app-/route/1-GET",
+					"aws:api_integration:my-app-/route/1-POST",
+					"aws:api_integration:my-app-/route/2-POST",
+					"aws:api_method:my-app-/-GET",
+					"aws:api_method:my-app-/route/1-GET",
+					"aws:api_method:my-app-/route/1-POST",
+					"aws:api_method:my-app-/route/2-POST",
+					"aws:api_resource:my-app-/route",
+					"aws:api_resource:my-app-/route/1",
+					"aws:api_resource:my-app-/route/2",
+					"aws:availability_zones:AvailabilityZones",
+					"aws:elastic_ip:my_app_0",
+					"aws:elastic_ip:my_app_1",
+					"aws:internet_gateway:my_app_igw",
+					"aws:load_balancer:my-app-tg",
+					"aws:load_balancer_listener:my-app-tg",
+					"aws:nat_gateway:my_app_0",
+					"aws:nat_gateway:my_app_1",
+					"aws:rest_api:api",
+					"aws:route_table:my_app_private0",
+					"aws:route_table:my_app_private1",
+					"aws:route_table:my_app_public",
+					"aws:subnet_private:my_app:my_app_private0",
+					"aws:subnet_private:my_app:my_app_private1",
+					"aws:subnet_public:my_app:my_app_public0",
+					"aws:subnet_public:my_app:my_app_public1",
+					"aws:target_group:tg",
+					"aws:vpc:my_app",
+					"aws:vpc_link:aws:load_balancer:my-app-tg",
+				},
+				Deps: []coretesting.StringDep{
+					{Source: "aws:api_integration:my-app-/-GET", Destination: "aws:api_method:my-app-/-GET"},
+					{Source: "aws:api_integration:my-app-/-GET", Destination: "aws:load_balancer:my-app-tg"},
+					{Source: "aws:api_integration:my-app-/-GET", Destination: "aws:rest_api:api"},
+					{Source: "aws:api_integration:my-app-/-GET", Destination: "aws:vpc_link:aws:load_balancer:my-app-tg"},
+					{Source: "aws:api_integration:my-app-/route/1-GET", Destination: "aws:api_method:my-app-/route/1-GET"},
+					{Source: "aws:api_integration:my-app-/route/1-GET", Destination: "aws:api_resource:my-app-/route/1"},
+					{Source: "aws:api_integration:my-app-/route/1-GET", Destination: "aws:load_balancer:my-app-tg"},
```

Comment:
do we know where that trailing dash on `my-app-` is coming from? It would be nice if we could get rid of it.

Reply:
its because we do appname-route. we could change that up if we want to though